$OpenBSD: patch-src_pdf_pdfextractoroutputdevice_cpp,v 1.2 2020/09/12 12:40:10 rsadowski Exp $

Starting with poppler-20, POPPLER_VERSION no longer is numeric -- it's
what POPPLER_VERSION_STRING was in earlier releases.

Index: src/pdf/pdfextractoroutputdevice.cpp
--- src/pdf/pdfextractoroutputdevice.cpp.orig
+++ src/pdf/pdfextractoroutputdevice.cpp
@@ -45,9 +45,7 @@ void PdfExtractorOutputDevice::drawImage(GfxState* sta
     pdfImg.d->m_refNum = ref->getRef().num;
     pdfImg.d->m_refGen = ref->getRef().gen;
 
-#if KPOPPLER_VERSION >= QT_VERSION_CHECK(0, 69, 0)
     pdfImg.d->m_colorMap.reset(colorMap->copy());
-#endif
     pdfImg.d->m_sourceHeight = height;
     pdfImg.d->m_sourceWidth = width;
     pdfImg.d->m_width = width;
